Cargo.toml

```diff
@@ -11,6 +11,8 @@ members = [
     "eax",
     "ocb3",
     "xaes-256-gcm",
+
+    "benches",
```

Comment:
`benches` are intentionally excluded from the workspace.

Reply:
You could either use a concrete git commit of the traits repo (before the inout changes) in the benches patch section, or you could temporarily disable the benches CI job.